cards/multilang-financial-professional-location/template.hbs

```diff
@@ -172,10 +174,11 @@
     {{#if ariaLabel}}aria-label="{{ariaLabel}}"{{/if}}>
     {{#if (any iconName iconUrl)}}
     <div class="HitchhikerCTA-iconWrapper">
-      <div class="HitchhikerCTA-icon" data-component="IconComponent" data-opts='{
-        "iconUrl": "{{iconUrl}}",
```

Comment:
This didn't have the handling for `relativePath`. Was this CTA broken then?

Reply:
just tested, yes unfortunately they were broken